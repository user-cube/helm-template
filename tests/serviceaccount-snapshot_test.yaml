# yaml-language-server: $schema=https://raw.githubusercontent.com/helm-unittest/helm-unittest/main/schema/helm-testsuite.json
suite: test serviceaccount snapshot
templates:
  - templates/serviceaccount.yaml
tests:
  - it: should match the snapshot with default values
    values:
      - ../src/values.yaml
    asserts:
      - matchSnapshot: {}
