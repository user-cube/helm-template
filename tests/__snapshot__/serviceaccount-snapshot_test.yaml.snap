should match the snapshot with default values:
  1: |
    apiVersion: v1
    automountServiceAccountToken: true
    kind: ServiceAccount
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: helm-template
        app.kubernetes.io/version: 1.16.0
        helm.sh/chart: helm-template-0.1.0
      name: RELEASE-NAME-helm-template
